packages/account/src/Types/common-prop.type.ts

```diff
@@ -184,21 +184,21 @@ export type TVerificationStatus = Readonly<
     Record<'none' | 'pending' | 'rejected' | 'verified' | 'expired' | 'suspected', string>
 >;
 
-type TDocumentList = Array<{
-    id: string;
-    text: string;
-    value?: string;
-    sample_image?: string;
-    example_format?: string;
-    additional?: {
-        display_name: string;
-        format: string;
-    };
-}>;
-
-export type TIDVFormValues = {
-    document_type: TDocumentList[0];
-    document_number: string;
-    document_additional?: string;
-    error_message?: string;
-};
+// type TDocumentList = Array<{
```

Comment:
is this comment expected? :)

Reply:
This is a head ache. I hope one of the refactor PR gets merged soon.